test/lib/ufe/testDeleteCmd.py

```diff
@@ -310,10 +305,342 @@ def testDeleteArgs(self):
         propsChildrenNames = childrenNames(propsChildren)
 
         self.assertNotIn(sphereShapeName, sphereChildrenNames)
-        self.assertNotIn(ball35Name, propsChildrenNames)
-        self.assertNotIn(ball34Name, propsChildrenNames)
+        self.assertIn(ball35Name, propsChildrenNames)
+
+    @unittest.skipUnless(ufeUtils.ufeFeatureSetVersion() >= 2, 'testDeleteArgs only available in UFE v2 or greater.')
+    def testDeleteArgs(self):
+        '''Delete Maya and USD objects passed as command arguments.'''
+
+        # Create our UFE notification observer
+        ufeObs = TestObserver()
+
+        if ufeUtils.ufeFeatureSetVersion() < 2:
+            ufe.Scene.addObjectDeleteObserver(ufeObs)
+            ufe.Scene.addObjectAddObserver(ufeObs)
+        else:
+            ufe.Scene.addObserver(ufeObs)
```

Comment:
You are skipping this test its not Ufe v2 or greater. So why do we need this condition? Line 317 will never be true - right?